cypress/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["es5", "dom"],
+    "types": ["cypress", "node"]
```

Comment:
I see you use cypress allure and cypress iframe plugins. Are you sure you have autocompletion for these commands in Visual Studio Code. You must provide them in "types" array to enable auto-completion.